source/includes/_tipser-react-elements.md

```diff
@@ -0,0 +1,263 @@
+# Tipser React Elements
+Tipser Elements is a React components library from Tipser enabling you to add shoppable content to your site with minimal effort.  You can add simple components like products with buy buttons, collections of products and articles, but also more complex grids or whole pages.
+
+The package allows you to use Tipser's product repository and create inline shops embedded into your page as well as the entire websites.
```

Comment:
This paragraph kind of repeats the last sentence of the previous paragraph. But I'm ok to leave it if you think it introduces some new information.

Reply:
Yes, I think I haven't focused on making this neat. That's the next step :)